pkg/release/release.go

```diff
@@ -30,6 +30,6 @@ var Image = map[string]string{
 	"ose_csi_node_registrar":    "quay.io/microshift/ose-csi-node-driver-registrar" + Base,
 	"ose_csi_livenessprobe":     "quay.io/microshift/ose-csi-livenessprobe" + Base,
 	"ovn_kubernetes_microshift": "quay.io/microshift/ovn-kubernetes-microshift:" + Base,
-	"pause":                     "quay.io/microshift/pause:" + Base,
+	"pod":                       "quay.io/microshift/pause:" + Base,
```

Comment:
If "pause" is not unique enough, can we make it into "pause_pod", not just "pod"?

Reply:
@ggiguash 'pod' is the exact image name in release payload, shall we keep as it is?

Reply:
I'm not sure. It's too generic for me, hence the comment. 
Anyway, let's see what other reviewers think.